packages/core/src/lib/telemetry.ts

```diff
@@ -132,43 +121,55 @@ function collectFieldCount (lists: Record<string, InitialisedList>) {
   return fields
 }
 
-function collectPackageVersions () {
-  const versions: Project['versions'] = {
-    '@keystone-6/core': '0.0.0', // effectively unknown
+async function collectPackageVersions () {
+  const packages: Project['packages'] = {
+    '@keystone-6/core': '0.0.0', // "unknown"
   }
 
-  for (const packageName of packageNames) {
+  for (const packageName of [
+    '@keystone-6/core',
+    '@keystone-6/auth',
+    '@keystone-6/fields-document',
+    '@keystone-6/cloudinary',
+    '@keystone-6/session-store-redis',
+    '@opensaas/keystone-nextjs-auth',
+  ]) {
     try {
-      const packageJson = require(`${packageName}/package.json`)
-      versions[packageName] = packageJson.version
+      const packageJson = await import(`${packageName}/package.json`)
+      packages[packageName] = packageJson.version
     } catch {
-      // do nothing, most likely because the package is not installed
+      // do nothing, the package is probably not installed
     }
   }
 
-  return versions
+  return packages
 }
 
 function printAbout () {
   console.log(`${y`Keystone collects anonymous data when you run`} ${g`"keystone dev"`}`)
   console.log()
-  console.log(`For more information, including how to opt-out see https://keystonejs.com/telemetry`)
 }
 
-export function printTelemetryStatus () {
-  const { telemetry } = getTelemetryConfig()
+function printNext (telemetry: Telemetry) {
+  if (!telemetry) {
+    console.log(`Telemetry data will ${r`not`} be sent by this system user`)
+    return
+  }
+  console.log(`Telemetry data will be sent the next time you run ${g`"keystone dev"`}`)
+}
 
+function printTelemetryStatus (telemetry: Telemetry) {
   if (telemetry === undefined) {
     console.log(`Keystone telemetry has been reset to ${y`uninitialized`}`)
```

Comment:
Maybe "is currently uninitialized" would be better wording here.

After resetting, subsequent `status` commands imply that an action has been taken, which is incorrect. The above covers a little better.

![image](https://github.com/user-attachments/assets/ab71b185-1809-4d0d-943f-33675d46c760)

Reply:
Resolved in 3dff2b2ee